dot_lpdev_cmds.sh

```diff
@@ -807,6 +807,8 @@ function __lpdev_wizard {
   #"Deposit tokens to node"
   "Start & set up verifier"
   "Update livepeer and cli"
+  "Install source-based dependencies"
```

Comment:
Can we make it say something like "Re-install ffmpeg" for now?  People might be confused about what "source-based dependencies actually entail...

Reply:
Indeed, it installs ffmpeg, x264 and nasm, and we may install other things later (FDK-AAC? Intel's media SDK?). But that's all in the service of ffmpeg, so we can go with a blanket simplification here. :+1: